functions/firebase/auth/app.rb

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START functions_firebase_auth]
+require "functions_framework"
+
+# Triggered by creation or deletion of a Firebase Auth user object.
+FunctionsFramework.cloud_event "hello_auth" do |event|
+  # The event parameter is a CloudEvents::Event::V1 object.
```

Comment:
Ditto to my comments in the `RTDB` PR - re-word this comment?

(Maybe re-word across any existing files in a separate PR, too? [Sorry for not catching this earlier!])

Reply:
Updated.